src/Compilers/VisualBasic/Portable/CodeGen/EmitExpression.vb

```diff
@@ -153,6 +156,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeGen
             EmitPopIfUnused(used)
         End Sub
 
+        Private Sub EmitComplexConditionalAccessReceiver(expression As BoundComplexConditionalAccessReceiver, used As Boolean)
```

Comment:
Generally we handle bound nodes in emit when the nodes are not otherwise representable in terms of other nodes or handling in Emit could result is noticeable better IL. It looks like this is expressible in the bound trees as "default(T) == null? refReceiver : valReceiver" . Perhaps it would be better to lower this just to reduce the number of node kinds that we need to handle in emit?